### DISCUSSION TITLE
[Analytical table]: [Whitespace in table towards right when column resized and pulled to left]

### DISCUSSION THREAD

**Discussion by kmanisha29** [UI5/webcomponents-react#6548] [Q&A] [answered]
https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/data-display-analyticaltable--default
When resize on columns is on, if we resize a column to left on inside, it leaves a whitespace towards right side.
If we resize it to right or expand, column further right goes out of page.  Is this an intended behaviour or bug?
is there a way to remove this behaviour and not leave whitespace or move column out of view?
<img width="939" alt="Screenshot 2024-10-23 at 3 17 51 PM" src="https://github.com/user-attachments/assets/075d80e6-e501-48da-87d9-9931d52b544b">
<img width="934" alt="Screenshot 2024-10-23 at 3 18 11 PM" src="https://github.com/user-attachments/assets/2fdec8f5-f41a-481c-a3bc-fefe4c3cf416">

**Comment by Lukas742** [comment, marked as answer]
Hi @kmanisha29 this behavior is by design and is not intended to be changed. Please see the explanation below:

>  if we resize a column to left on inside, it leaves a whitespace towards right side.

The table container width should always stay the same, as otherwise UI elements could shift if a column is resized.

> If we resize it to right or expand, column further right goes out of page.

Without this functionality, it would be complicated increasing the width of the last column.
